### DISCUSSION TITLE
how to open multiple tabs in single editor

### DISCUSSION THREAD

**Discussion by tom-cloyd** [VSCodium/vscodium#932] [Q&A] [answered]
I'm baffled. Sometimes when I click a file in the explorer it opens in a new tab in an editor and sometimes it replaces an existing tab. I have been able to find documentation about this or any explanation anywhere.

How can I reliably open a file in a NEW tab?

**Comment by paulcarroty** [comment, marked as answer]
Use double click in the file explorer.

**Reply by tom-cloyd** [reply]
Thank you - but I object to this as a path to multiple tabs, for reasons of accessibility. I'm a musician and I have fast fingers, but I still have trouble issuing that double-click fast enough to get the desired response every time. For some folks who have muscular impairments of various sorts, this double-click business just won't work. I think Ctrl+ click is a much better keystroke combination to associate with this multiple-tab outcome.

**Reply by paulcarroty** [reply]
Another way is focus on file then press `Enter`. Dig deeper, I'm sure you can customize it.

**Reply by tom-cloyd** [reply]
That works nicely! Thanks again for your help. I WILL dig deeper and see what I can learn. After years of devotion to the jEdit editor, I have moved to vscodium and I'm way beyond delighted. This is far and away my favorite computer-supported tool.

**Comment by daiyam** [comment]
@tom-cloyd Try with the settings `"workbench.editor.enablePreview": false`. It will disable the preview so you won't need to double-click to open a file in a new tab.
